microsoft/botframework-solutions
# fix calendar bug

## Description
fix some bugs

## Related Issue
1. state won't clear when update meeting again
2. some ambiguous wordings in update and delete flow
3. only show the start date when show the cross-days meeting
4. when the meeting title contains '"', rendering card will go wrong.
5. won't show the meetings are past when ask "show meetings from 4pm to 9pm"

## Testing Steps
1. input "update my 5pm meeting to 6pm" -> "cancel" -> "update my meeting"
bot should ask for time again
2. input "update my 5pm meeting"(you don't have meeting at 5pm)
bot shouldn't say "It seems there is no event with that name, try again please."
input "update my test meeting"(you don't have meeting that's title is "test")
bot shouldn't say "It seems there is no event at this time, please try again."
3. create a cross-days meeting and ask bot to show it. the adaptive card will show the start date and end date
4. create a meeting with a title contains '"', then ask bot to show it.
5. input "show meetings from 4pm to 9pm"(current time is after 4pm), bot will show all meetings in 4-9pm.
